### DISCUSSION TITLE
Posizionamento termostato

### DISCUSSION THREAD

**Discussion by fmulassano** [indomus/forum#1825] [GENERALE - Discussioni generiche]
il termostato attualmente è vicino alla porta di ingresso a circa 1mt più o meno dal termosifone dell'ingresso.
Contatore e quadro sono sopra questo termosifone. Per comodità potrei mettere il termostato vicino al quadretto ma poi il termosifone lo disturberebbe? falserebbe la lettura?

Dalla porta di ingresso poi c'è un corridoio di 6mt prima di arrivare in sala.
Conto di mettere le valvole wifi su tutti i termosifoni.

Cosa mi consigliate di fare? Mettendo le valvole, il termostato può stare ovunque?

**Comment by Scialla** [comment]
Se non hai la caldaia con il ricircolo interno, lascia un termosifone privo di valvola smart, altrimenti la caldaia può andare in blocco

**Reply by fmulassano** [reply]
grazie
